libp2p/Host.h

```diff
@@ -50,38 +51,38 @@ namespace p2p
 
 class Host;
 
-enum class Origin
-{
-	Unknown,
-	Self,
-	SelfThird,
-	PerfectThird,
-	Perfect,
-};
-
-struct Node
+struct PeerInfo
 {
 	NodeId id;										///< Their id/public key.
-	unsigned index;									///< Index into m_nodesList
+	
+	// p2p: move to NodeIPEndpoint
```

Comment:
Re: This comment and others. This is an interim PR for a work-in-progress. Small steps -- this won't be merged into develop. Hopefully tomorrows code will be merged into develop, though :)